nextstrain/cli/runner/__init__.py

```diff
@@ -264,6 +266,14 @@ def run(opts: Options, working_volume: NamedVolume = None, extra_env: Env = {},
     if opts.__runner__ is singularity and opts.image is docker.DEFAULT_IMAGE: # type: ignore
         opts.image = singularity.DEFAULT_IMAGE # type: ignore
 
+    if envdirs := os.environ.get("NEXTSTRAIN_RUNTIME_ENVDIRS"):
+        try:
+            opts.envdir = [
+                *[DirectoryPath(d) for d in envdirs.split(os.pathsep) if d],
+                *opts.envdir ]
```

Comment:
Is there a reason to prepend here and not `opts.envdir.extend`?

Reply:
Yep. It follows the traditional Unix cascade of: built-in default → environment variable → command line option. So for example, if you provide an env var `A` via `NEXTSTRAIN_RUNTIME_ENVDIRS` and also again via `--envdir`, the value from the latter will be used.